api/server/server.go

```diff
@@ -1557,3 +1564,11 @@ func ServeApi(job *engine.Job) error {
 
 	return nil
 }
+
+func toBool(str string) bool {
```

Comment:
mmm haven't reallly looked at that. But I firstly thought about moving this to pkg/stringutils but it's not so generic to convert a string to a bool that way so I skipped that...then as lk4d4 said I moved it here so if someone other needs it (and she will because many bool flags are passed to daemon that way) she may reuse it.. I think this is the new implementation of Env.GetBool that server should use to convert string to bool maintaining backward compatibility... to my knowledge of course...